detect_variants/detect_variants.cwl

```diff
@@ -17,14 +17,14 @@ inputs:
         secondaryFiles: .bai
     interval_list:
         type: File[]
-    strelka_config:
-        type: File
     dbsnp_vcf:
         type: File?
         secondaryFiles: .tbi
     cosmic_vcf:
         type: File?
         secondaryFiles: .tbi
+    strelka_exome_mode:
```

Comment:
Are boolean required args or can this be optional with a default of false? Maybe I'm missing the cwl syntax. At the detect_variants.cwl level should we have a less specific `--exome` boolean arg instead of it being strelka specific? Currently strelka is the only caller that has different usage/inputs, but I wouldn't be surprised if others, ex. pindel, also have different workflows/usage based on this flag.